packages/nodes-base/nodes/Google/Task/GenericFunctions.ts

```diff
@@ -0,0 +1,83 @@
+import { OptionsWithUri } from 'request';
+
+import {
+	IExecuteFunctions,
+	IExecuteSingleFunctions,
+	ILoadOptionsFunctions
+} from 'n8n-core';
+
+import { IDataObject } from 'n8n-workflow';
+
+export async function googleApiRequest(
+	this: IExecuteFunctions | IExecuteSingleFunctions | ILoadOptionsFunctions,
+	method: string,
+	resource: string,
+	body: any = {},
+	qs: IDataObject = {},
+	uri?: string,
+	headers: IDataObject = {}
+): Promise<any> {
```

Comment:
As the code is written now, you might want to add in ` // tslint:disable-line:no-any` to get rid of the tslint errors associated with `<any>`